app/dashboard/views.py

```diff
@@ -5586,7 +5586,7 @@ def fulfill_bounty_v1(request):
     if payout_type == 'fiat' and not fulfiller_identifier:
         response['message'] = 'error: missing fulfiller_identifier'
         return JsonResponse(response)
-    elif (payout_type == 'qr' or payout_type == 'polkadot-ext') and not fulfiller_address:
+    elif payout_type in ['qr', 'polkadot-ext', 'binance-ext'] and not fulfiller_address:
```

Comment:
nice 🌈

Reply:
Oh wait i think this should be  `polkadot_ext`and `binance_ext`!
Derp derp typo mistake